disparate-flies/BangazonSite
# Homepage

Please structure your pull requests this way:

- **Purpose of PR:**  The home page now shows the latest 20 products. When you click on the title, you are able to see the details of that specific item.


- **Issue number this request impacts:** #4 


- **Description of what was changed:** home controller accesses the database to allow view of the products. The products are displayed in descending order. The index view inside the home folder iterates over the products title and has a hyperlink that allows you to view the details of the product.


- **Thorough list of steps to test**

1.  Checkout to master
2.  Type git fetch
3. cd my branch "homepage"
4. Type start Bangazon.sln
5. Open the file name HomeController
6. press  play "IIS Express" button at the top
7. When the program opens up in your browser, you should see the list of approx. 20 or less products under the title "Latest Products"
8. Run an insert sql statement into products (you will leave out the productId and the DateCreated bc they are computer generated - you will need to get the long UserId code from your local database).
(ex: INSERT INTO Product (Description, Quantity, Title, Price, UserId, City, ProductTypeId)
VALUES (
	'Computer',
	200,
	'MacBook Pro',
	1000,
	'415b0fa4-4801-438a-aace-50da1cbb0d9c',
	'Franklin',
	2);
9. Now that you have add an extra product, rebuild the application and the product you just added should show at the top of the product list on the home page.
10. Click on the title of a product, the details will be displayed
11. Run the Unit Test to make sure everything passes on your computer

if you don't do this you will be
**FIRED**
